tools/webpack-core/src/plugins/CopyBentleyStaticResourcesPlugin.ts

```diff
@@ -74,26 +74,36 @@ export class CopyBentleyStaticResourcesPlugin extends AbstractAsyncStartupPlugin
 
   public async runAsync(compiler: Compiler) {
     const paths = getPaths();
-    const bentleyDir = path.resolve(paths.appNodeModules, "@bentley");
-    let subDirectoryNames: string[];
-    try {
-      subDirectoryNames = await fs.readdir(bentleyDir);
-    } catch (err: any) {
-      this.logger.error(`Can't locate ${err.path}`);
-      return;
-    }
-    for (const thisSubDir of subDirectoryNames) {
-      if (!(await isDirectory(path.resolve(bentleyDir, thisSubDir))))
-        continue;
-
-      const fullDirName = path.resolve(bentleyDir, thisSubDir);
-      for (const staticAssetsDirectoryName of this._directoryNames) {
-        await tryCopyDirectoryContents(
-          path.join(fullDirName, "lib", staticAssetsDirectoryName),
-          this._useDirectoryName ? compiler.outputPath : path.join(compiler.outputPath, staticAssetsDirectoryName),
-        );
+
+
+    const copyContents = async (basePath: string) => {
+      let subDirectoryNames: string[];
+      try {
+        subDirectoryNames = await fs.readdir(bentleyDir);
+      } catch (err: any) {
+        this.logger.error(`Can't locate ${err.path}`);
+        return;
+      }
+      for (const thisSubDir of subDirectoryNames) {
+        if (!(await isDirectory(path.resolve(bentleyDir, thisSubDir))))
+          continue;
+
+        const fullDirName = path.resolve(bentleyDir, thisSubDir);
+        for (const staticAssetsDirectoryName of this._directoryNames) {
+          await tryCopyDirectoryContents(
+            path.join(fullDirName, "lib", staticAssetsDirectoryName),

```

Comment:
I gotta make sure my changes to delivering to both cjs and esm sub dir won't break anything